docs/kh2/index.md

```diff
@@ -9,7 +9,7 @@
   
   * [IDX/IMG](file/type/idx.md)
   * [BAR](file/type/bar.md)
-  * [ARD](file/type/ard.md)
+  * [ARD](file/type/areadata.md)
```

Comment:
Shouldn't rng.md already be added to the index.md or are you waiting until it has useful information?

Reply:
Waiting for it to be actually useful